lexicons/app/bsky/graph/getSuggestedFollowsByActor.json

```diff
@@ -24,6 +24,11 @@
                 "type": "ref",
                 "ref": "app.bsky.actor.defs#profileView"
               }
+            },
+            "isFallback": {
```

Comment:
I think I'm inclined to use a positive here rather than a negative. Like `isPersonalized` which would be set to `true` if so and `false` if not. 

Couple reasons:
- generally, i like to avoid negative variable names when there's not a good reason not to
- specific to this case, i'm trying to think of the baseline functionality here & make that the default. It seems more natural to me that the baseline would be not-personalized and the "special case" is personalized

Reply:
Not overly concerned about this fwiw if you disagree or if there might be compatibility issues.

Reply:
Good points here. This endpoint's baseline actually is personalized though, and the special case is to fall back to generic suggestions if Why can't compute cluster-based results.

Given that, how do you feel about the naming?

Reply:
(followed up in DMs, let's stick with `isFallback`)